benchmarks/tpch/create_tables.sql

```diff
@@ -0,0 +1,133 @@
+-- Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Should this also contain the disclaimer about being derived from TPC-H since it is performing DDL? Don't know the answer there?

Reply:
Good point. I will go ahead and add this.